iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/IpRestrictedAdminAccessCheck.java

```diff
@@ -35,10 +39,33 @@
   public static final int SSH_PORT = 22;
   public static final int RDP_PORT = 3389;
 
+  private static final Pattern PORT_RANGE_PATTERN = Pattern.compile("^(?<from>\\d{1,5})-(?<to>\\d{1,5})$");
```

Comment:
The PORT_RANGE_PATTERN is whitespace sensitive - a single space anywhere within the matched string will result in <no_match>. Are we certain this is the intended behaviour?

(This is the only potentially issue-raising comment.)

Reply:
Has to be tested by AppSec. Everything on our side would be guessing.